modules/RTC/TraceablePeerConnection.js

```diff
@@ -1907,6 +1903,24 @@ TraceablePeerConnection.prototype.getConfiguredVideoCodec = function() {
     return defaultCodec;
 };
 
+/**
+ * Returns the codecs in the current order of preference as configured on the peerconnection.
+ *
+ * @returns {Array}
+ */
+TraceablePeerConnection.prototype.getConfiguredVideoCodecs = function() {
+    const sdp = this.peerconnection.localDescription?.sdp;
+
+    if (!sdp) {
+        return [];
+    }
+    const parsedSdp = transform.parse(sdp);
+    const mLine = parsedSdp.media.find(m => m.type === MediaType.VIDEO);
```

Comment:
Can there be a case in which there is no video m line?

Reply:
Nope, `offerToReceiveVideo` guarantees that they will always be a video m-line.